java/src/org/certificatetransparency/ctlog/serialization/Deserializer.java

```diff
@@ -114,12 +115,12 @@ public static ParsedLogEntry parseLogEntry(InputStream merkleTreeLeaf, InputStre
 
     if (entryType == Ct.LogEntryType.X509_ENTRY) {
       Ct.X509ChainEntry x509EntryChain = parseX509ChainEntry(extraData,
-          treeLeaf.timestampedEntry.signedEntry.getX509());
+          treeLeaf.timestampedEntry.signedEntry.x509);
       logEntry.x509Entry = x509EntryChain;
     } else if (entryType == Ct.LogEntryType.PRECERT_ENTRY) {
       Ct.PrecertChainEntry preCertChain = parsePrecertChainEntry(extraData,
-          treeLeaf.timestampedEntry.signedEntry.getPrecert());
-       logEntry.precertEntry = preCertChain;
+          treeLeaf.timestampedEntry.signedEntry.preCert);
+      logEntry.precertEntry = preCertChain;
```

Comment:
In C++, I'd have been sorely tempted to use `auto`. In Go, you'd do `localThing := getThing()`. Considering Java is the one where users are most likely to have a fancy IDE (and so, need this the least), as well as typically having the longest type names, I find this especially masochistic?

I guess that's why they invented the fancy IDEs that can rename all the things for you? :wink: